bird.cpp

```diff
@@ -39,10 +39,14 @@ void Bird::draw()
 {
 	glDepthMask(GL_FALSE);
 
-	glm::mat4 identity = glm::mat4(1.0f);
-	glm::mat4 result = identity;
+	// Pitch (forward tilt in planespeak) calculation.
+	float pitch = (float) atan(velocity.y)*5000; // TODO Trial-and-error value
```

Comment:
I'm not entirely sure what you mean. Remark so far though:
Because `velocity.x` is not used yet, the angle cannot be calculated right because there is only up-and-down movement -> the angle is either 90 degrees or -90 degrees.